common/lib/capa/capa/capa_problem.py

```diff
@@ -62,6 +74,9 @@
 # main class for this module
 
 
+DEFAULT_QUESTION_TEXT = _("You must specify meaningful question text.")
```

Comment:
@lamagnifica This is the default text that we will show for every problem, in case if user does not provide question text or if they don't use the proper formatter to identify the title. Do you think this make sense ?